store/team.go

```diff
@@ -103,22 +90,36 @@ func (es *teamstore) GetTeamByToken(token string) (*haaukins.Team, error) {
 
 func (es *teamstore) SaveTeam(t *haaukins.Team) error {
 	es.m.Lock()
+	fmt.Println("AAAAAAAAAAAAAAAAAAAAAAA inside save team function TeamStore")
+	//todo save the team in the DB
+
 
 	email := t.Email()
 	_, ok := es.emails[email]
 	if ok {
 		es.m.Unlock()
 		return ErrEmailAlreadyExists
-	}else{
-		es.emails[t.Email()] = t.ID()
 	}
+	_, err := es.dbc.AddTeam(context.Background(), &pbc.AddTeamRequest{
+		Id:                   t.ID(),
+		EventTag:             string(es.eConf.Tag),
+		Email:                t.Email(),
+		Name:                 t.Name(),
+		Password:             t.GetHashedPassword(),
+	})
+	if err != nil {
+		es.m.Unlock()
+		return err
+	}
+
 	es.emails[email] = t.ID()
 	es.teams[t.ID()] = t
 	es.m.Unlock()
 
 	return nil
 }
 
+//todo does it create a token or it save it in teamstore? Ahmet. I thinnk it saves it. so i can change hte name of this function
 func(es *teamstore) CreateTokenForTeam (token string, in *haaukins.Team) error {
```

Comment:
Yeah, the name of the function does not fully represent what it is doing. It just saves the existing given token to teamstore.